assets/stylesheets/dark.qss

```diff
@@ -196,6 +196,18 @@ QScrollBar::sub-line {
     border: none;
 }
 
+QLineEdit {
+    color: rgb(255, 255, 255);
+    background-color: transparent;
+    border-width: 1px;
```

Comment:
i actually did something similar for the tabbar

you can simplify this with:
`border-bottom: 2px solid palette(base);`

Reply:
I attached pictures. Your way doesn't work when the style only includes the bottom border.

Reply:
to clarify, `border-bottom` doesn't seem to work with QLineEdit.

Reply:
seems to work like this:
light
````
QLineEdit {
    border-top-left-radius: 6px;
    border-top-right-radius: 6px;
    border-bottom-left-radius: 1px;
    border-bottom-right-radius: 1px;
    border: none;
    padding: 12px;
    color: rgba(0, 0, 0, 222);
    background-color: rgba(0, 0, 0, 10);
    border-bottom: 1px solid rgb(0, 0, 0, 108);
}

QLineEdit:focus {
    background-color: rgba(0, 0, 0, 20);
    border-bottom: 2px solid palette(base);
}
````

dark
```
QLineEdit {
    border-top-left-radius: 6px;
    border-top-right-radius: 6px;
    border-bottom-left-radius: 1px;
    border-bottom-right-radius: 1px;
    border: none;
    padding: 12px;
    color: rgb(255, 255, 255);
    background-color: rgba(255, 255, 255, 10);
    border-bottom: 1px solid rgba(255, 255, 255, 136);
}

QLineEdit:focus {
    background-color: rgba(255, 255, 255,20);
    border-bottom: 2px solid palette(base);
}
```

Reply:
I don't understand how the combinations of properties interact with each other. I tried using `border-bottom` without all of that other stuff and it didn't work.

Reply:
Regardless, this feature needs to be merged in and subsequent refinements balanced against feature improvements.

Reply:
well the `border-*-radius` ones were just for stylistic purposes, but basically `border: none` would remove the default styling, and then you can set the border style you want (in this case `border-bottom`)

I would argue that the style of the QLineEdit should be added in this feature, since it'll be introducing it into `develop`